src/test/java/org/ays/auth/controller/AysRoleEndToEndTest.java

```diff
@@ -535,9 +531,20 @@ void givenValidRoleUpdateRequest_whenPermissionsUpdatedAndNameUnchanged_thenRetu
 
         // Given
         String id = role.getId();
+        List<AysPermission> newPermissions = List.of(
+                new AysPermissionBuilder()
+                        .withValidValues()
+                        .withId("17dd50f6-61fe-4a30-a136-d9b80649e7fe")
```

Comment:
End to end testlerde hard codded ID'ler vermemeye çalışalım çünkü ID değişebilir, rol silinebilir buradaki testin aslında onunla ilgili olmaması gerekiyor. Bu sebeple veri tabanından alınan rollerden birini alıp kullanabiliriz
```suggestion
                        .withId("17dd50f6-61fe-4a30-a136-d9b80649e7fe")
```